pages/guide.py

```diff
@@ -568,6 +568,13 @@ def process_post_data(self, feature_id, stage_id=0):
           'experiment_extension_reason')
     if self.touched('ongoing_constraints'):
       feature.ongoing_constraints = self.form.get('ongoing_constraints')
+    
+    # Add user who updated to list of editors.
+    # TODO(danielrsmith): This should be removed when enabling new permissions.
+    email = self.get_current_user().email()
+    if (email not in feature.editors and email not in feature.owner and
```

Comment:
Let's also check that the user is not already a site editor.  Ideally the whole condition would be `if not permissions.can_edit_feature(user, feature.id)`, but that would be calling the version of `can_edit_feature()` that is still in your pending PR #2017 that needs to wait a while because we don't want the new permission rules to be enforced yet.

Maybe you could copy over the `can_edit_feature()` method from your pending PR, so long as you temporarily give it a new name, like `strict_can_edit_feature()`.

Reply:
Yes, that's a good idea. I've changed it so that admins and site editors will not get added as feature editors if they make changes to a specific feature 👍